src/index.css

```diff
@@ -55,18 +96,17 @@
 @import '/src/common/assets/fonts/font.css';
 
 ::-webkit-scrollbar {
-  width: 0.5em;
-  height: 0.5em;
+  width: 3px;
```

Comment:
Can we use tailwind? Please also update the color, border...

Reply:
yes.